invenio_records_ui/templates/invenio_records_ui/tombstone.html

```diff
@@ -0,0 +1,7 @@
+{%- extends config.RECORDS_UI_BASE_TEMPLATE %}
+
+{%- block page_body %}
+TOMBSTONE
```

Comment:
The template needs to be created (like-wise for ``detail.html``). This template basically just ensures that ``record`` and ``pid`` is available in the template context.